web/static/actor/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Actor Info</title>
+    <script type="module" src="/ux/curio-ux.mjs"></script>
+    <script type="module" src="/actor-summary.mjs"></script>
+</head>
+<body style="visibility:hidden" data-bs-theme="dark">
+<curio-ux>
+    <actor-info></actor-info>
+</curio-ux>
+<script>
```

Comment:
This being defined after the tag is used doesn't work, should be a separate file